.github/workflows/windows.yml

```diff
@@ -3,6 +3,54 @@ name: Windows GitHub CI
 on: [pull_request, push]
 
 jobs:
+  shared:
+    runs-on: windows-latest
+    # Run a job for each of the specified target architectures:
+    strategy:
+      matrix:
+        arch:
+          - win64
+          - win32
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ilammy/msvc-dev-cmd@v1
+      with:
+        arch: ${{ matrix.arch }}
+    - uses: ilammy/setup-nasm@v1
+      with:
+        platform: ${{ matrix.arch }}
+    - name: prepare the build directory
+      run: mkdir _build
+    - name: config
```

Comment:
How about this?

``` yaml
    - name: config for ${{ matrix.arch }}
```

Reply:
Why, you can see the matrix.arch in the build name anyway: https://github.com/openssl/openssl/pull/15349/checks?check_run_id=2638029715